packages/nextjs/next.config.js

```diff
@@ -3,6 +3,12 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
+  typescript: {
+    ignoreBuildErrors: process.env.NEXT_PUBLIC_IGNORE_BUILD_ERROR === "true" ? true : false,
```

Comment:
you can do it without ternary :) `ignoreBuildErrors: process.env.NEXT_PUBLIC_IGNORE_BUILD_ERROR === "true"`

Reply:
Trust me I actually spent 4-5 min trying to figure out the best way to write this and this didn't clicked 😂😂, Thanks @rin-st looks better fixed at #197